manifests/params.pp

```diff
@@ -17,4 +17,5 @@
 
   $java_initial_heap_size = '1g'
   $java_max_heap_size = '1g'
+  $java_opts = '1g'
```

Comment:
Shouldn't this default be an empty string?

Reply:
Sorry, you are right. I fixed it.